tests/group_tree.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import sunbeam
+
+
+class TestGroupTree(unittest.TestCase):
+    def setUp(self):
+        norne = '../../examples/data/norne/NORNE_ATW2013.DATA'
+        self.es = sunbeam.parse(norne, ('PARSE_RANDOM_SLASH', sunbeam.action.ignore))
+
+    def test_group(self):
+        gr = self.es.schedule.group(timestep=2)['PROD']
+
+        self.assertEqual('PROD', gr.name)
+        self.assertEqual('MANI-B1', gr.children[0].name)
+        self.assertEqual(6, len(gr.children))
+        self.assertEqual('FIELD', gr.parent.name)
+        self.assertEqual(2, gr.timestep)
+
+    def test_timestep_groups(self):
+        for group in self.es.schedule.groups(timestep=3):
+            for child in group.children:
+                self.assertIsNotNone(child.name)
+
+        group = self.es.schedule.group(timestep=3)['PROD']
+        children = ['MANI-B1', 'MANI-B2', 'MANI-D1', 'MANI-D2', 'MANI-E1', 'MANI-E2']
+        for child in group.children:
+            self.assertIn(child.name, children)
+
+    def test_grouptree(self):
+        tree = self.es.schedule.grouptree[0]
+        parent = tree['MANI-B1']['parent']
+        children = tree['MANI-B1']['children']
```

Comment:
We don't want `['parent']` just `.parent` will do.

Reply:
Ja, kan kanskje fjerne hele 
 'schedule.grouptree' da nevnte funksjonalitet er implementert i 'schedule.group'